torch/ao/quantization/observer.py

```diff
@@ -1092,15 +1092,14 @@ def _adjust_min_max(
         # Therefore, we avoid such division operation.
         downsample_rate = int(
             torch.ceil(
-                (combined_max - combined_min) * upsample_rate / (self.max_val - self.min_val)
+                ((combined_max - combined_min) / (self.max_val - self.min_val)) * upsample_rate
```

Comment:
I'm not sure if changing these orders will cause any issues for existing tests, might be good to import to fbcode and make sure CI is green